tidb-cloud/data-service-api-key.md

```diff
@@ -0,0 +1,51 @@
+---
+title: API Key in Data Service
+summary: Learn how to create, edit, and delete an API key for a Data App.
+---
+
+# API Key in Data Service
+
+The TiDB Cloud Data API uses [HTTP Digest Authentication](https://en.wikipedia.org/wiki/Digest_access_authentication). It protects your private key from being sent over the network. For more details about HTTP Digest Authentication, refer to the [IETF RFC](https://datatracker.ietf.org/doc/html/rfc7616).
+
+## API key overview
+
+- The API key contains a public key and a private key, which act as the username and password required in the HTTP Digest Authentication. The private key only displays upon the key creation.
+- The API key belongs to a Data App and is used to access the data in the TiDB Cloud clusters.
+- You must provide the correct API key in every request. Otherwise, the TiDB Cloud responds with a `401` error.
+
+## API key management
+
+The following sections describe how to create, edit, and delete an API key for a Data App.
+
+### Create an API key
+
+To create an API key for a Data App, perform the following steps:
+
+1. Navigate to the [**Data Service**](https://tidbcloud.com/console/dataservice) page of your project.
+2. Click the name of your target Data App to go to its details page.
+3. In the **API Key** section, click **Create API Key**.
+4. In the **Create API Key** dialog box, enter a description for your API key, and then click **Next**. The private key and public key are displayed.
+
+    Make sure that you have copied and saved the private key in a secure location. The private key only displays upon creation. After leaving this page, you will not be able to get the full private key again.
+
+5. Click **Done**.
+
+### Edit an API key
+
+To edit an API key for a Data App, perform the following steps:
+
+1. Navigate to the [**Data Service**](https://tidbcloud.com/console/dataservice) page of your project.
+2. Click the name of your target Data App to go to its details page.
+3. In the **API Key** section, click **Edit** in the API key row that you want to change.
+4. You can update the description of the API key.
+5. Click **Update**.
+
+### Delete an API key
+
```

Comment:
Do we add a note here to tell users to ensure that the target app no longer uses this API key?

Reply:
https://github.com/pingcap/docs/pull/12711/commits/3705a41eab19e875732b890537bf3a90557672d1#diff-08c9ef46df79a9dc16a0c5bc178efd0f230ea3b2851e6b262f0eed91d0a02ae2R45-R47

```markdown
> **Note:**
>
> Before you delete an API key, make sure that the API key is not used by any Data App.
```